osgp/shared/osgp-ws-admin/src/main/resources/schemas/common-ws-admin.xsd

```diff
@@ -15,6 +15,12 @@
     </xsd:restriction>
   </xsd:simpleType>
 
+  <xsd:simpleType name="Email">
```

Comment:
Ik zou de typename ook veranderen naar iets algemeens, bijv. 'UserName'. Er is nu geen sprake meer van specifiek een email-adres.
Geldt ook voor de andere `.xsd` files